projects/demo/src/modules/components/multi-select/examples/1/index.ts

```diff
@@ -9,7 +9,7 @@ import {TuiDataListWrapper} from '@taiga-ui/kit';
 import {TuiMultiSelectModule, TuiTextfieldControllerModule} from '@taiga-ui/legacy';
 
 const ITEMS: readonly string[] = [
-    'Luke Skywalker',
+    'Luke Skywalker and a long time ago in a galaxy far, far away..',
```

Comment:
nit: looks like one example of a long tag is enough for the test

Reply:
fixed